pages.pt_BR/common/7za.md

```diff
@@ -8,18 +8,30 @@
 
 `7za a {{arquivo_compactado.7z}} {{caminho/arquivo_ou_diretório}}`
 
+- Criptografar um arquivo existente (incluindo cabeçalhos):
+
+`7za a {{arquivo_criptografado.7z}} -p{{senha}} -mhe=on {{arquivo.7z}}`
+
 - Descompactar um arquivo mantendo a estrutura de diretórios original:
 
 `7za x {{arquivo_compactado.7z}}`
 
-- Compactar utilizando um tipo específico de arquivamento/compressão:
+- Descompactar um arquivo em um diretório específicado pelo usuário:
 
-`7za a -t{{zip|gzip|bzip2|tar}} {{arquivo_compactado.7z}} {{caminho/arquivo_ou_diretório}}`
+`7za x {{arquivo.7z}} -o{{caminho/diretório}}`
 
-- Exibir os tipos de arquivamento/compressão disponíveis:
+- Descompactar um arquivo para a saída padrão:
 
-`7za i`
+`7za x {{path/to/archive.7z}} -so`
+
+- Compactar utilizando um tipo específico de arquivamento/compressão:
+
+`7za a -t{{zip|gzip|bzip2|tar}} {{arquivo_compactado.7z}} {{caminho/arquivo_ou_diretório}}`
 
 - Exibir o conteúdo de um arquivo:
 
 `7za l {{arquivo.7z}}`
+
+- Exibir os tipos de arquivamento/compressão disponíveis:
```

Comment:
For Romance languages, the present indicative fits better.
```suggestion
- Exibe os tipos de arquivamento/compressão disponíveis:
```
Please apply this to the other examples.